### PR TITLE
Fix squeeze arg in transform of Potato

### LINKED ISSUE #66: Potato issue: Type Error!
I'm getting a strange issue with potato!

```
TypeError: 'numpy.float64' object does not support item assignment
```

image

Any ideas?

### PR DESCRIPTION
**Pull request by qbarthelemy**
Fixes #66

### CODE REVIEW

**Review by agramfort — DISMISSED** [review]

**Review by agramfort — COMMENTED** [review]
Actually can you add a tiny test?

**Comment by agramfort** [comment]
I've now setup GH actions.

can you rebase on master after updating a test? thx @qbarthelemy

File: tests/test_clustering.py [hunk 1]
```diff
@@ -79,9 +79,11 @@ def test_Potato_init():
 
     # transform
     pt.transform(covset)
+    pt.transform(covset[0][np.newaxis, ...]) # transform a single trial
 
     # predict
     pt.predict(covset)
+    pt.predict(covset[0][np.newaxis, ...]) # predict a single trial
```
Comment (qbarthelemy) [inline tests/test_clustering.py:86]: This line tests the case reported by #66

**Comment by agramfort** [comment]
Thx @qbarthelemy

**Pull request merged**